packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/common_services/user_account_service.dart

```diff
@@ -41,4 +39,25 @@ class LoginUseCase {
 
     return true;
   }
+
+  Future<bool> logout() async {
+    // Unsubscribe user push token
+    try {
+      final pushToken =
+          await _pushSubscriptionRepository.getToken(vapidKey: webVapidKey);
+      if (pushToken != null) {
+        await _pushSubscriptionRepository.unsubscribe(pushToken);
+      }
+    } catch (e) {
+      print(e);
```

Comment:
I believe here we need to use log instead of print

Reply:
Fixed